src/test/java/com/lucidworks/spark/SolrRelationTest.java

```diff
@@ -80,6 +81,12 @@ public void testMultiValuedExport() throws Exception {
       assertTrue(row.length() > 5);
       assertTrue(fromSolr.count() == 100);
 
+      {
+        Dataset df = sparkSession.read().format(Constants.SOLR_FORMAT()).options(options).option(ConfigurationConstants.SOLR_FIELD_PARAM(), "out_clicks_ss").load();
```

Comment:
This should be using 'export' handler with sort field as 'id' even though the `fields` option is provided